packages/client/src/app.module.ts

```diff
@@ -35,7 +35,7 @@ import { ServiceWorkerModule } from '@angular/service-worker';
         UserModule,
         UiModule,
         ServiceWorkerModule.register('ngsw-worker.js', {
-            enabled: !isDevMode(),
+            enabled: true,
```

Comment:
Is there a reason why this setting was changed?

Reply:
Yes. The service worker module needs to be enabled if we want to create push notifications. If we disabled it in dev mode we won't be able to test.

Reply:
Hum ok, this was implemented like this because it causes problem when developing with this. Because it caches everything, so we can't see the changes. Do you think there is a way we can fix this?

Reply:
Since I need the service workers to be enabled to be able to send a notification, I didn't find a solution, but I didn't find may problems with caching either.
Since the notifications are not a feature that will change very often, a possible fix is to revert it to `!isDevMode()` and to update the documentation to change it to `true`when we want to work in the notification feature.

Reply:
Good, sounds like a plan!